src/pages/ResetPassword.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+
+import api from '../api'
+import styled from 'styled-components'
+
+const Title = styled.h1.attrs({
+    className: 'h1',
+})``
+
+const Label = styled.label`
+    margin: 5px;
+`
+
+const InputText = styled.input.attrs({
+    className: 'form-control',
+})`
+    margin: 5px;
+    width: 30%;
+`
+
+const Wrapper = styled.div.attrs({
+    className: 'form-group',
+})`
+    margin: 30px 80px;
+`
+
+const Button = styled.button.attrs({
+    className: `btn btn-primary`,
+})`
+    margin: 15px 15px 15px 5px;
+`
+
+function ResetPassword() {
+    const [email, setEmail] = useState('');
+
+    async function handleResetPassword() {
```

Comment:
Nice) Also please convert function to useCallback - [usecallback](https://uk.reactjs.org/docs/hooks-reference.html#usecallback)